cpp/src/parquet/file_writer.h

```diff
@@ -90,8 +92,12 @@ class PARQUET_EXPORT RowGroupWriter {
    */
   int64_t num_rows() const;
 
+  /// \brief total uncompressed bytes written by the page writer
   int64_t total_bytes_written() const;
+  /// \brief total bytes still compressed but not written
```

Comment:
Should we explain it can only exist in the buffered row group?

Reply:
Sure, I can write code to verify this.